app/models/member.rb

```diff
@@ -29,7 +31,8 @@ def name=(full_name)
 
   def liquid_data
     full_name = name
-    attributes.merge({
+    attributes.symbolize_keys.merge({
```

Comment:
attributes comes out with string keys, and I want to overwrite one of those keys in this merge. I'd either have to use string keys in the lines below this or I need to symbolize them. happy to do either